README.md

```diff
@@ -10,13 +10,13 @@ It is a free culture work, licensed under Creative Commons Attribution Share-Ali
 
 ## Repository Currently Outdated
```

Comment:
Should this part be clarified to mention that pull requests are likely not accepted (for example https://github.com/ethereum/yellowpaper/pull/876), unless they completely specify a network upgrade?  That's why this section suggests applying for a grant?